components/brave_vpn/resources/panel/components/select-region-list/style.ts

```diff
@@ -80,37 +88,29 @@ export const RegionContainer =
   background: ${color.container.background};
 
   ${(p) =>
-    p.selected
+    p.fillBackground &&
+    (p.selected
       ? css`
           border: 1px solid ${color.divider.interactive};
           background: ${color.container.interactive};
           box-shadow: ${effect.elevation['01']};
-
-          leo-button {
-            display: none;
-          }
         `
       : css`
           &:hover {
             background: ${color.container.highlight};
             box-shadow: ${effect.elevation['01']};
-
-            leo-button {
-              opacity: 1;
-              transition: opacity 0.3s;
-            }
           }
+        `)}
 
-          &:has(:focus) leo-button, &:focus-visible leo-button {
-            opacity: 1;
-          }
-        `}
+  & > :last-child {
+    border-bottom: none;
+  }
 `
 
-export const RegionConnect = styled(Button)`
+export const RegionConnect = styled(Button)<{ right: string }>`
   opacity: 0;
   position: absolute;
-  right: ${spacing.xl};
+  right: ${(p) => p.right};
```

Comment:
nit: might be better to pull the hover styling into this component, so everything is in one place.


```suggestion
  right: ${(p) => p.right};
  transition: opacity 0.3s ease-in-out;

  ${RegionCountry}:hover &, ${RegionCity}:hover & { 
    opactity: 1;
  }
```

`&` is a placeholder for the selector for the current component